MAINTAINERS.md

```diff
@@ -1,10 +1,13 @@
+# OpenAPI Specification Editors
```

Comment:
This list has served as the record of TSC membership, though I think that might be made explicit? By adding this title, it also changes the framing. For example, while TSC members are tasked with evolving the spec, that includes managing issues, encouraging community, etc.

Reply:
Added title to express original purpose of TSC membership